### ISSUE TITLE
Extension does not display on tabs that start with chrome-extension://

### ISSUE THREAD

**Issue by Pascal666** [babyman/quick-tabs-chrome-extension#340]
All my other extensions show up on these tabs, but not Quick Tabs.  This is really annoying because most of my tabs are suspended by The Great Suspender so most of my tabs do not display the Quick Tabs icon.  I have to find a tab that is not suspended in order to use it.

**Comment by babyman** [comment]
That's weird, I use The Great Suspender and don't see this issue.  But I am using Brave:

![image](https://user-images.githubusercontent.com/120736/106367647-2bc7af00-6312-11eb-8ae6-b17d7fd9444f.png)

and have the following key mapping:

![image](https://user-images.githubusercontent.com/120736/106367669-44d06000-6312-11eb-85ba-705f1e64a1ae.png)

I'm not sure what you mean by not showing the Quick Tabs icon, you mean this icon in the nav is hidden when you have some tabs (but not all) in focus?

![image](https://user-images.githubusercontent.com/120736/106367706-7ea16680-6312-11eb-8149-50712bfa35a2.png) 

Evan

**Comment by Pascal666** [comment]
<img width="313" alt="QuickTabs" src="https://user-images.githubusercontent.com/1330708/106368079-38450b00-6304-11eb-8ac0-692b69208797.png">
<img width="312" alt="QuickTabsMissing" src="https://user-images.githubusercontent.com/1330708/106368081-38450b00-6304-11eb-93e0-d9da4bd92cbd.png">
Chrome Version 87.0.4280.141 (Official Build) (64-bit)

Quick Tabs 2020.9.8

**Comment by babyman** [comment]
That's really weird, and you have it pinned?

![image](https://user-images.githubusercontent.com/120736/106371188-67bd3d00-632f-11eb-844f-992708ed770c.png)

Does the shortcut key to launch the extension work on tabs with the chrome-extension:// url?

**Comment by Pascal666** [comment]
Yes, it is pinned.  Unpinning it removes it from all tabs.  Repinning it and moving it back next to my URL bar restores the original behavior I reported above.

Yes, the shortcut brings it up on all tabs, although it uses a very interesting animation that makes it look like the icon comes out from behind my TGS icon.

I found the problem.  It looks like Chrome has a maximum number of extensions it will display icons for, if you go over this limit it removes the icon to the right of the URL bar, and tabs that start with chrome-extension:// support one less icon than other tabs.  I disabled one of my extensions that I don't use very often and now Quick Tabs shows up on all tabs.

Apologies for wasting your time and thank you for your assistance.
